<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinaMaaritys</class>
 <widget class="QWidget" name="MinaMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Käyttäjätiedot</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nimi</string>
       </property>
       <property name="buddy">
        <cstring>nimiEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nimiEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sähköposti</string>
       </property>
       <property name="buddy">
        <cstring>emailEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Puhelinnumero</string>
       </property>
       <property name="buddy">
        <cstring>phoneEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="phoneEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="tunnistatumisGroup">
     <property name="title">
      <string>Tunnistautuminen</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="kayta2fa">
        <property name="text">
         <string>Kaksivaiheinen tunnistautuminen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>378</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="salasanaNappi">
        <property name="text">
         <string>Vaihda salasana</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/lukittu.png</normaloff>:/pic/lukittu.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nakymaGroup">
     <property name="title">
      <string>Näkymä</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="normaaliRadio">
        <property name="text">
         <string>Kitsaan tilaaja tai kokeilija</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/Possu64.png</normaloff>:/pic/Possu64.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="toffeeRadio">
        <property name="text">
         <string>Tilitoimistokäyttäjä tai tilitoimiston asiakas</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/pixaby/toimisto.svg</normaloff>:/pic/pixaby/toimisto.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tilausGroup">
     <property name="title">
      <string>Tilaus</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="tilausInfo">
        <property name="text">
         <string>Tilausinfo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>550</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="tilausButton">
          <property name="text">
           <string>Tee tilaus</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/freeicons/timantti.svg</normaloff>:/freeicons/timantti.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="laskutusButton">
          <property name="text">
           <string>Laskutustiedot</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/mail.png</normaloff>:/pic/mail.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>182</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nimiEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>kayta2fa</tabstop>
  <tabstop>salasanaNappi</tabstop>
  <tabstop>normaaliRadio</tabstop>
  <tabstop>toffeeRadio</tabstop>
  <tabstop>tilausButton</tabstop>
  <tabstop>laskutusButton</tabstop>
 </tabstops>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
